src/textual/app.py

```diff
@@ -510,7 +503,7 @@ async def on_event(self, event: events.Event) -> None:
         # Handle input events that haven't been forwarded
         # If the event has been forwarded it may have bubbled up back to the App
         if isinstance(event, events.Mount):
-            view = View()
+            view = View(id="_root")
```

Comment:
Why the id of _root ?

Reply:
Think this is just a leftover from debugging. I was printing out the node IDs from the code that walks the DOM tree and was wondering why one of them was printing `None`. Turns out it was the this view, which looks like the root of the tree.